### DISCUSSION TITLE
Совместимость с ASAR модами

### DISCUSSION THREAD

**Discussion by swzxu** [cucumber-sp/yandex-music-linux#115] [Q&A] [answered]
Можно как то заводить моды для яндекс музыки для винды на этом клиенте?

**Comment by cucumber-sp** [comment, marked as answer]
На данный момент те моды что я видел, вручную меняют исходный код. Если есть моды представленные в виде патчей, то тогда, конечно, их можно скомбинировать с нашими патчами
